pkg/plan/apply.go

```diff
@@ -205,11 +207,27 @@ func (p *Plan) syncReleasesKubedog(kubedogConfig *kubedog.Config) error {
 	defer cancel()
 
 	// KubeInit
-	err = kube.Init(kube.InitOptions{})
+	kubeconfigPath := os.Getenv("KUBECONFIG")
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfigPath)
 	if err != nil {
 		return fmt.Errorf("failed to initialize kubernetes config: %w", err)
 	}
 
+	if config != nil {
+		clientset, err := kubernetes.NewForConfig(config)
+		if err != nil {
+			return fmt.Errorf("failed to initialize kubernetes config: %w", err)
+		}
+		kube.Kubernetes = clientset
+		kube.Client = clientset
+
+		dynamicClient, err := dynamic.NewForConfig(config)
+		if err != nil {
+			return fmt.Errorf("failed to initialize kubernetes config: %w", err)
+		}
+		kube.DynamicClient = dynamicClient
+	}
+
```

Comment:
lets move this to separate function (probably in pkg/helper)

Reply:
done